Jenkinsfile

```
@@ -144,17 +144,19 @@ pipeline {
         /**
         Packages Artifacts & Publishes release
         */
-        stage('Package&Publish') {
+        stage('Package&Publish-Snapshot') {
           agent { label 'linux && immutable && debian-11' }
           options { skipDefaultCheckout() }
           environment {
             PATH = "${env.PATH}:${env.WORKSPACE}/bin"
             HOME = "${env.WORKSPACE}"
+            WORKFLOW = "snapshot"
           }
           when {
             beforeAgent true
             allOf {
               expression { return params.release_ci }
+              expression { params.build_type != 'staging' }
```

I think it's a pretty fragile design to check for `not staging` for the snapshot stage (and vice versa).
```suggestion
              expression { params.build_type == 'snapshot' || params.build_type == 'snapshot+staging' }
```
What happens the next time you add another build type? with the current design the `new-build-type` will result with building all three types.

---

There are only 2 build types in the DRA process. The twice-a-day snapshot & staging which is designated for build candidates.